# Shipment contents message
# This structure contains the information about a shipment being submitted

# The ID of the shipping_box or tray the shipment was detected in
string destination_id

string station_id

# Collection of products
DetectedProduct[] products
